src/custodia/log.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, plugin, debug):
         extra = {'origin': plugin.origin}
         super(CustodiaLoggingAdapter, self).__init__(logger, extra=extra)
 
+    # pylint: disable=arguments-differ
     def exception(self, msg, *args, **kwargs):
```

Comment:
So what happen when pythoin2 code calls this ?

Reply:
All is fine, it's just cosmetics.

The signature is different in Python 2 than Python 3. pylint is complaining about the difference. We cannot use the Python 3 signature because that is invalid syntax in Python 3. In Python 2, keywords after ``*args`` is not supported: ``exception(self, msg, *args, exc_info=True, **kwargs)``